app/gate/api.js

```diff
@@ -1,4 +1,5 @@
 import axios from 'axios'
+import { isEmpty } from 'lodash'
```

Comment:
Because of tree shaking, there is two way to add this kind of functions to the project.

  1. install directly the `lodash.isEmpy` library
  2. importing like: `import isEmpty from 'lodash/isEmpty';

Reply:
this is done
thanks